api/types/accesslist/convert/v1/accesslist.go

```diff
@@ -125,14 +135,24 @@ func ToProto(accessList *accesslist.AccessList) *accesslistv1.AccessList {
 		}
 	}
 
+	ownership := accesslistv1.Inclusion_INCLUSION_EXPLICIT
+	if enumVal, ok := toInclusionProto(accessList.Spec.Ownership); ok {
```

Comment:
nit: do we really need `ok` value here ? 

the `toInclusionProto` can just return Inclusion_INCLUSION_EXPLICIT in case of unknown input value